packages/unlock-js/openapi.yml

```
@@ -1779,6 +1779,89 @@ paths:
         500:
           $ref: '#/components/responses/500.InternalError'
 
+  /v2/api/users/{emailAddress}/sendVerificationCode:
```

Like I already asked, please do NOT use camel case for URLS.
```suggestion
  /v2/api/users/{emailAddress}/send-verification-code:
```